spiceaidocs/docs/data-connectors/s3.md

```diff
@@ -31,13 +31,14 @@ Example: `name: cool_dataset`
 
 - `endpoint`: The S3 endpoint, or equivalent (e.g. MinIO endpoint), for the S3-compatible storage. E.g. `endpoint: https://my.minio.server`
 - `region`: Region of the S3 bucket, if region specific. E.g. `region: us-east-1`
+- `timeout`: Timeout for S3 operations. E.g. `timeout: 60s`
```

Comment:
Also specify what the defaults are for each parameter.

Reply:
Done